ttam-linters/deps-checker.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+declare -A ERROR_CODES
+
+# declaring an hastable for file and its codes
+ERROR_CODES["requirements.txt"]="A001"
+ERROR_CODES["Pipfile"]="A002"
+ERROR_CODES["Dockerfile"]="A003"
+ERROR_CODES["package.json"]="A004"
+ERROR_CODES["production.txt"]="A005"
+
+# File passed by superlinter
+FILE_PATH=$1
+
+# This one is to get file name
+FILE_NAME=$(echo "${FILE_PATH}" | rev | cut -d "/" -f1 | rev)
+
+# Checking if files are start with "-r"
+POINTER_FILES="$(grep -iE "^-r " "${FILE_PATH}")"
+
+format_output_for_lintly() {
+
+  output=${FILE_PATH}":1:1:${ERROR_CODES[${FILE_NAME}]} does not explicitly point to Artifactory,"
```

Comment:
Should we hardcode the name `Artifactory` or make it more general and say "does not explicitly point to an approved package repository" or something?

Reply:
Yes that make sense. Will update.